src/ansi-c/parser.y

```diff
@@ -101,6 +101,7 @@ extern char *yyansi_ctext;
 %token TOK_NE        "!="
 %token TOK_ANDAND    "&&"
 %token TOK_OROR      "||"
+%token TOK_RANGE     ".."
```

Comment:
Syntax extensions are always a bit controversial.  Is there an ACSL construct you could use?